cmd/operator-sdk/new/cmd.go

```diff
@@ -341,6 +349,14 @@ func doHelmScaffold() error {
 		return fmt.Errorf("new helm scaffold failed: (%v)", err)
 	}
 
+	crdsDir := filepath.Join(projectName, scaffold.CRDsDir)
+	gcfg := genutil.Config{InputDir: crdsDir, OutputDir: crdsDir}
+	crd := gencrd.NewNonGo(gcfg, *resource)
+	if err := crd.Generate(); err != nil {
+		return errors.Wrapf(err, "error generating CRD for %s", resource)
+	}
+	log.Info("Generated CustomResourceDefinition manifests.")
+
```

Comment:
The above code shows  duplicated see: 264 to 271. So, if is duplicated,  wdyt about centralized in a func?